docs/errors.md

```diff
@@ -0,0 +1,61 @@
+---
+sidebar_position: 5
+title: Error Codes
+description: Find more information on various error codes the bot can provide and what to do about it.
+---
+import ErrorEmbed from '@site/src/components/Embeds/ErrorEmbed';
+
+## Table
+| Code | Name                        | Guide                                 |
+|------|-----------------------------|---------------------------------------|
+| 1    | SUGGESTION_MESSAGE_DELETED  | [click](#suggestion-message-deleted)  |
+| 2    | MISSING_PERMISSIONS         | [click](#missing-permissions)         |
+| 3    | MISSING_SUGGESTIONS_CHANNEL | [click](#missing-suggestions-channel) |
+| 4    | MISSING_LOG_CHANNEL         | [click](#missing-log-channel)         |
+| 5    | SUGGESTION_NOT_FOUND        | [click](#suggestion-not-found)        |
+| 6    | OWNER_ONLY                  | [click](#owner-only)                  |
+| 7    | SUGGESTION_CONTENT_TOO_LONG | [click](#suggestion-content-too-long) |
+| 8    | INVALID_GUILD_CONFIG_CHOICE | [click](#invalid-guild-config-choice) |
+| 9    | COMMAND_ON_COOLDOWN         | [click](#command-on-cooldown)         |
+| 10   | GENERIC_FORBIDDEN           | [click](#generic-forbidden)           |
+| 11   | UNHANDLED_ERROR             | [click](#unhandled-error)             |
+| 12   | GENERIC_NOT_FOUND           | [click](#generic-not-found)           |
+
+## Error Information
+
+### Suggestion Message Deleted
+This error is thrown when attempting to manage a suggestion which it's corresponding message has been deleted.
+<ErrorEmbed title="Lorem ipsum" description="Hello world" code={1} />
+
+### Missing Permissions
+This error is thrown when the bot is missing permissions to perform an action.
```

Comment:
This is raised when the bot is missing permissions to perform an action **and** knows what permissions are missing. The bot will include the required permissions in the error message.